lib/rex/post/meterpreter/client_core.rb

```diff
@@ -781,7 +788,8 @@ def negotiate_tlv_encryption(timeout: client.comm_timeout)
 
     {
       key:  sym_key,
-      type: key_type
+      type: key_type,
+      is_weak_key: is_weak_key
```

Comment:
The preferred Ruby style for something like this I think would be to use the `?` suffix instead of the `is_` prefix.

```suggestion
      weak_key?: !is_weak_key
```

Reply:
Using the !is_weak_key will invert the logic? or am I wrong

Reply:
You're right, I was editing this and toying with the idea of calling it `secure_key?: !is_weak_key`. Good catch, I've updated my suggestion.